### PR TITLE
show SV new genotype

### PR DESCRIPTION
**Pull request by hanars**

### CODE REVIEW

**Comment by ShifaSZ** [comment]
It hasn't implemented all the requested features. For example, it isn't displaying/filtering `new` variants. Right?

**Comment by hanars** [comment]
>It hasn't implemented all the requested features. For example, it isn't displaying/filtering new variants. Right? 

It has, that was all implemented in the pipeline to work with the existing functionality we already built for gCNV, and I confirmed that it is working for this new SV data

File: seqr/utils/elasticsearch/constants.py [hunk 1]
```diff
@@ -426,6 +426,7 @@ def get_prediction_response_key(key):
     'prev_call': {'format_value': bool},
     'prev_overlap': {'format_value': bool},
     'new_call': {'format_value': bool},
+    'prev_num_alt': {'format_value': lambda i: None if i is None else int(i)},
```
Comment (ShifaSZ) [inline seqr/utils/elasticsearch/constants.py:429]: The type of `prev_num_alt` is an integer in the ES. Do we still need to format it?

File: ui/shared/components/panel/variants/VariantIndividuals.jsx [hunk 2]
```diff
@@ -291,6 +298,21 @@ const Genotype = React.memo(({ variant, individual, isCompoundHet, genesById })
     previousCall = { content: 'Identical Call', hover: 'Identical call in previous callset', color: 'blue' }
   } else if (genotype.prevOverlap) {
     previousCall = { content: 'Overlapping Call', hover: 'Overlapping call in previous callset', color: 'teal' }
+  } else if (Number.isInteger(genotype.prevNumAlt)) {
+    const hasSameHemiXGenotype = isHemiX && (genotype.numAlt === 1 || genotype.numAlt === 2) &&
+      (genotype.prevNumAlt === 1 || genotype.prevNumAlt === 2)
```
Comment (ShifaSZ) [inline ui/shared/components/panel/variants/VariantIndividuals.jsx:303]: I need clarification about the logic for the same hemiX genotype. Does it mean it is a hemiX with the same `numAlt` and `prevNumAlt` values?

File: ui/shared/components/panel/variants/VariantIndividuals.jsx [hunk 3]
```diff
@@ -291,6 +298,21 @@ const Genotype = React.memo(({ variant, individual, isCompoundHet, genesById })
...
+    previousCall = {
+      content: 'New Genotype',
+      color: 'olive',
+      hover: (
+        <span>
+          Previous callset:
+          <AlleleContainer>{svNumAltGenotype(genotype.prevNumAlt, isHemiX)}</AlleleContainer>
+          {/* eslint-disable-next-line react/jsx-one-expression-per-line */}
+          {hasSameHemiXGenotype && <i>Hemi alt allele change: {genotype.prevNumAlt} to {genotype.numAlt}</i>}
```
Comment (ShifaSZ) [inline ui/shared/components/panel/variants/VariantIndividuals.jsx:312]: The variable name `hasSameHemiXGenotype` looks like something changed, but the displaying text says `allele changed`.

File: seqr/utils/elasticsearch/constants.py (hunk 1 above)
Comment (hanars) [reply seqr/utils/elasticsearch/constants.py:429]: yes if format is omitted it defaults to string, which is wrong

File: ui/shared/components/panel/variants/VariantIndividuals.jsx (hunk 2 above)
Comment (hanars) [reply ui/shared/components/panel/variants/VariantIndividuals.jsx:303]: For hemiX num alt of 1 and 2 are displayed the exact same so to the user they would be unable to see that anything had changed, so this is checking for that case

File: ui/shared/components/panel/variants/VariantIndividuals.jsx (hunk 3 above)
Comment (hanars) [reply ui/shared/components/panel/variants/VariantIndividuals.jsx:312]: the number of called alt alleles has changed

**Review by ShifaSZ — APPROVED** [review]
LGTM.

**Pull request merged**